src/test/java/seedu/sourcecontrol/model/ModelManagerTest.java

```diff
@@ -68,34 +68,34 @@ public void setGuiSettings_validGuiSettings_setsGuiSettings() {
 
     @Test
     public void setAddressBookFilePath_nullPath_throwsNullPointerException() {
```

Comment:
```suggestion
    public void setSourceControlFilePath_nullPath_throwsNullPointerException() {
```
should we change the method names too? same for the other methods in this file